perf/src/bin/perf_client.rs

```diff
@@ -16,54 +16,54 @@ use perf::stats::{OpenStreamStats, Stats};
 use std::path::PathBuf;
 
 /// Connects to a QUIC perf server and maintains a specified pattern of requests until interrupted
-#[derive(StructOpt)]
-#[structopt(name = "client")]
+#[derive(Parser)]
+#[clap(name = "client")]
 struct Opt {
     /// Host to connect to
-    #[structopt(default_value = "localhost:4433")]
+    #[clap(default_value = "localhost:4433")]
```

Comment:
Looks like the only place we you use string instead of `SocketAddr`, might be good to format in later prs

Reply:
`SocketAddr` cannot represent hostnames.